### ISSUE TITLE
Ore Excavation with Hammers doesn't work as intended

### ISSUE THREAD

**Issue by BakermanLP** [MinecraftModDevelopmentMods/BaseMetals#117]
Hello there,

I wanted to use your newest builds of BaseMetals and some other mods. But there is an issue with the mod OreExcavation. 

Somewhere at this part of the mod BaseMetals there should be something different, that OreExcavation will work as intended:
https://github.com/MinecraftModDevelopmentMods/BaseMetals/blob/fbcc67a4ee46fe4c63a4e1ace63a9ee7bed03113/src/main/java/com/mcmoddev/lib/item/ItemMetalCrackHammer.java#L76

Some information about the necessary changes can be found here:
https://github.com/Funwayguy/OreExcavation/issues/6

Sorry, that I am not able to explain it better.

Michael (aka BakermanLP)

**Comment by dshadowwolf** [comment]
The Crack Hammer, despite appearances, is not a mining tool. It is a relatively cheap crusher/macerator/SAG Mill/etc... that uses Steve/Alex power and not RF/EU/etc... 

Do note, however, that we are calling the base function in ItemTool, which should do what you are saying needs done. Also note that, with the dev versions (which we are trying to get to a release-able state) dropping a stack of something in the world and right-clicking it will process as much of the stack as the hammer has durability.

Basically... This seems, to me, like a non-issue as the hammer (in dev versions) has means to do what you're asking built-in, just in a slightly different manner than you are expecting. Other than that... we are doing nothing that should stop the event from propogating - indeed, we are calling into ItemTool for its handler to run.
